### DISCUSSION TITLE
Enable emoji chooser in comments

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3030] [Feature Requests]
I'm imagining something similar to the icon interface, but emoji... and it is pulled-up when you click the emoji icon in the New Comment component.

**Comment by niklasgrewe** [comment]
@benhaynes I do have one small point... Would it be possible to use and select my 
 own Icons Packs also in Directus?

For example, you could add icons to your items (stored in the database as text) and then embed them on your website with SVG Sprite or as an icon fonts. This way you would not have to use the Media Library for icons.

**Comment by benhaynes** [comment]
This issue is specifically for internal usage (on Item Comments) — but you could easily duplicate the current Icon interface and adapt it to server/save your custom emoji/icon set. You can pretty much do anything you want with custom interfaces.

**Comment by Nitwel** [comment]
This is hidden now until we have time to implement a proper emoji interface. See https://github.com/directus/next/issues/691#issuecomment-712445639

The question also is if we want to have such interface because windows has a emoji picker build in via `win + .` and I guess mac also has one?

**Comment by benhaynes** [comment]
I still think we want it, it's very common within most chat and comment systems, and many users don't know about the system shortcuts. I also think having a good way to allow for extensible third-party icons is a nice benefit to this interface.

**Comment by nickrum** [comment]
This has since been added 👍
